lib/rubocop/cop/performance/inefficient_hash_search.rb

```diff
@@ -42,7 +42,10 @@ class InefficientHashSearch < Cop
         PATTERN
 
         def on_send(node)
-          add_offense(node, message: msg(node)) if inefficient_include?(node)
+          return unless inefficient_include?(node)
+          return if node.children[0].receiver.nil?
+
+          add_offense(node, message: msg(node))
```

Comment:
Perhaps you could also rename `#msg` -> `#message` and remove the argument from the call to `add_offense`? 🙂